pupil_src/shared_modules/calibration_choreography/base_plugin.py

```diff
@@ -140,7 +141,10 @@ def is_user_selection_for_gazer_enabled(cls) -> bool:
 
     @classmethod
     def default_selected_gazer_class(cls):
-        return cls.user_selectable_gazer_classes()[0]
+        if Gazer3D_v1x in cls.user_selectable_gazer_classes():
+            return Gazer3D_v1x
```

Comment:
```suggestion
        if default_gazer_class in cls.user_selectable_gazer_classes():
            return default_gazer_class
```

Reply:
Use default_gazer_class instead of explicit Gazer3D_v1x